onedal/common/tests/test_policy.py

```diff
@@ -47,6 +47,9 @@ def test_with_numpy_data(queue):
 @pytest.mark.parametrize("queue", get_queues("cpu,gpu"))
 @pytest.mark.parametrize("memtype", get_memory_usm())
 def test_with_usm_ndarray_data(queue, memtype):
+    if queue is None:
+        pytest.skip("queue=None not supported for usm_ndarray")
+
```

Comment:
There may be a bug here we need to investigate.

Reply:
I think disabling this not good solution. What kind of bug here?
It should work with queue=None and/or None data_queue for `_get_policy`

Reply:
This yields a mismatch and fails, if you would like to change this PR to a bugfix, let me know.

Reply:
The issue is on DPCTL side, a None queue will look for a cached queue, which will often be the GPU causing the failure.  The get_policy works fine, test is malformed for None. I.e. people should be very careful with direct usm_ndarray buffer allocations with memory

Reply:
I don't think that this is bug.
`dpctl` just differently processes None input. I am good with the changes here. Thank you!